libvmi/driver/kvm/kvm.c

```diff
@@ -277,6 +285,31 @@ parse_seg_reg_value(
     return (reg_t) strtoll(ptr, (char **) NULL, 16);
 }
 
+static addr_t
+parse_mtree(char *mtree_output)
+{
+    char *ptr;
+
+    if(NULL == (ptr = strstr(mtree_output, "alias ram-above-4g")))
+    goto error;
+
+    for(int i = 0; i <= 50;i++) {
```

Comment:
why `50` ?

Reply:
If I remember correctly, I ran into some weird off-by-one after a configuration change when I was doing a static offset. Probably don't need to rewind the pointer by that much.